Documentation/kubelet-wrapper.md

```diff
@@ -74,6 +74,17 @@ ExecStart=/usr/lib/coreos/kubelet-wrapper \
   ...other flags...
 ```
 
+### Allow pods to use rbd volumes
+
+Pods using the [rbd volume plugin][rbd-example] to consume data from ceph must ensure that the kubelet has access to modprobe. Add the following options to the `RKT_OPTS` env before launching the kubelet via kubelet-wrapper:
+
+```ini
```

Comment:
I waffled on this because I didn't want people to copy and paste this over existing RKT_OPTS. But, I think it makes sense to be consistent with the other examples and hint at where you might find a RKT_OPS env being set in the first place. Updated to reflect this.